srcpkgs/nitrokey-udev-rules/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'nitrokey-udev-rules'
+pkgname=nitrokey-udev-rules
+version=1.0.0
+revision=1
+build_style=meta
```

Comment:
this is not a meta package, it has files

Reply:
Sorry. It seems obvious now that you've mentioned it.